encoding/mkv.md

```diff
@@ -0,0 +1,505 @@
+chaps02 = [(0, 1677, 'Part A'), (3837, 'Part B'), (17239, 'Middle Card'), (17359, 30977, 'Part C'), (33111, 'Part D'), (33663, 34022, 'Preview')]
+chaps03 = [(0, 1941, 'Part A'), (4101, 'Part B'), (13668, 'Middle Card'), (13788, 30473, 'Part C'), (32609, 'Part D'), (33663, 34022, 'Preview')]
+chaps04 = [(0, 2565, 'Part A'), (4723, 'Part B'), (18724, 'Middle Card'), (18844, 30975, 'Part C'), (33109, 'Part D'), (33661, 34020, 'Preview')]
+chaps05 = [(0, 839, 'Part A'), (2997, 'Part B'), (15058, 'Middle Card'), (15178, 31265, 'Part C'), (33399, 'Part D'), (33663, 34022, 'Preview')]
+chaps06 = [(0, 1701, 'Part A'), (3861, 'Part B'), (16976, 'Middle Card'), (17096, 30256, 'Part C'), (32392, 'Part D'), (33662, 34021, 'Preview')]
+chaps07 = [(0, 1102, 'Part A'), (3261, 'Part B'), (21698, 'Middle Card'), (21818, 31217, 'Part C'), (33351, 'Part D'), (33663, 34022, 'Preview')]
+chaps08 = [(0, 3932, 'Part A'), (6090, 'Part B'), (16543, 'Middle Card'), (16664, 30448, 'Part C'), (32584, 'Part D'), (33662, 34021, 'Preview')]
+chaps09 = [(0, 719, 'Part A'), (2878, 'Part B'), (11581, 'Preview'), (11701, 31000, 'Part C'), (33134, 'Part D'), (33662, 34021, 'Preview')]
+chaps10 = [(0, 791, 'Part A'), (2949, 'Part B'), (16448, 'Middle Card'), (16568, 30160, 'Part C'), (32296, 'Part D'), (33662, 34021, 'Preview')]
+chaps11 = [(0, 287, 'Part A'), (2446, 'Part B'), (18892, 'Middle Card'), (19012, 29081, 'Part C'), (31216, 'Part D'), (33661, 34020, 'Preview')]
+# chaps12 = (0, 34212, 'Part A') --> already cut with eztrim
+chaps13 = [(2158, 'Part 1 - "You Never Let Us Down"'), (16374, 'Middle Card I'), (16494, 'Part 2 - "Always Growing Closer"'), (17797, 'Middle Card II'), (17941, 'Part 3 - "Let\'s Change You Into This!"'), (27171, 'Middle Card III'), (27291, 30473, 'Part 4 - "I\'m Your Big Sister"')]
+```
+
+Once we have these lists,
+we will call `audio_trim` with them and their respective files.
+
+```py
+import acsuite as ac
+ac.audio_trim('ep01.m2ts', chaps01)
+ac.audio_trim('ep02.m2ts', chaps02)
+...
+ac.audio_trim('ep11.m2ts', chaps11)
+ac.audio_trim('ep13.m2ts', chaps13)
+```
+
+Running the above script with Python (may take a few minutes to complete)
+will result in *many* new files in your working directory.
+
+For the author's example, the resulting files and their sizes are:
+
+- `ep1_cut.wav` -- `ep13_cut.wav` (total: 2.9 GiB)
+- `(ed|op|nced|ncop)_cut.wav` (total: 65.6 MiB)
+
+[acshelp]: https://github.com/OrangeChannel/acsuite/blob/a7e36aa65533a93bbaaa75b6fc5345194b27a5df/acsuite.py#L127
+
+
+## Encoding the audio
+
+We will be using FFmpeg's *libopus* and *flac* encoders.
+The following examples simply loop[^2] the command for every `ep*.wav` file,
+and all four op/ed files.
+
+For the main episodes, we will be using opus:
+
+```sh
+$ fd -e wav --regex "(_cut)" -E 'op*' -E 'ed*' -E 'nc*' -x ffmpeg -i {} -c:a libopus -b:a 128k -cutoff 20000 {.}.opus
```

Comment:
You don't do anything with the capture and `--regex` is the default, so just use `fd .e wav '_cut' …`. Besides, if you exclude `*op*` and `*ed*` you can skip the nc exclude.